### PR TITLE
Implement append behavior as the default for variable (tick) data type

### PR DESCRIPTION
**Pull request by Notargets**

### CODE REVIEW

File: executor/writer.go [hunk 1]
```diff
@@ -149,68 +154,87 @@ type IndirectRecordInfo struct {
...
+	fp.Seek(primaryOffset, stdio.SeekStart)
 	_, err = fp.Write(obuf)
-	if err != nil {
-		return err
-	}
+	return err
+}
 
-	return nil
+func insertionSort(data []byte, rLen int32) {
```
Comment (rocketbitz) [inline executor/writer.go]: still needed?

File: executor/writer.go [hunk 2]
```diff
@@ -149,68 +154,87 @@ type IndirectRecordInfo struct {
...
+			return err
+		}
+		dataToBeWritten = append(oldData, dataToBeWritten...)
+		dataLen = currentRecInfo.Len + dataLen
+	}
 	/*
-		If this is a continuation write, we adjust the targetRecInfo accordingly
+		Sort the data by the timestamp to maintain on-disk sorted order
+	 */
+	//insertionSort(dataToBeWritten, varRecLen)
```
Comment (rocketbitz) [inline executor/writer.go]: here too?

File: executor/writer.go (hunk 1 above)
Comment (Notargets) [reply executor/writer.go]: no!

File: executor/writer.go (hunk 2 above)
Comment (Notargets) [reply executor/writer.go]: no again!

**Pull request merged**